packages/api/src/command/webhook/webhook.ts

```diff
@@ -193,10 +197,12 @@ export const sendPayload = async (
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
 ): Promise<any> => {
   try {
+    const hmac = crypto.createHmac("sha256", apiKey).update(JSON.stringify(payload)).digest("hex");
     const res = await axios.post(url, payload, {
       headers: {
         "x-webhook-key": apiKey,
```

Comment:
One last concern. @leite08 had mentioned getting rid of sending "the key with locked box", but removing this would make the change a breaking change. So not sure if its worth the headache right now. We could wait on it

Reply:
thoughts @leite08 @Goncharo?

Reply:
Discussed in standup - will leave it for now but notify customers that in a round a month will be removing "x-webhook-key" header and they should migrate.

Reply:
That sounds good!

Can we create a ticket to remove it later, please?